### PR TITLE
{math}[GCCcore/10.2.0] makeinfo v6.7, GDB v10.1, MPC v1.2.1, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GDB/GDB-10.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('texinfo', '6.7'),
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GDB/GDB-10.1-GCCcore-10.2.0.eb:17]: Does GDB need all of `texinfo` to build? If it only requires `makeinfo` then we can build that directly instead, which is simpler than the `texinfo` install (see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/makeinfo/makeinfo-6.7-GCCcore-9.3.0.eb).

Comment (Flamefire) [reply easybuild/easyconfigs/g/GDB/GDB-10.1-GCCcore-10.2.0.eb:17]: You are right, makeinfo is enough, thanks!

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c3659a5618ffd89f8c74f541f7c1a228 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0305u05a.bear.cluster - Linux RHEL 8.2, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/159f03dda016f22302d26ce11333b1c3 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Going in, thanks @Flamefire!

**Pull request merged**